### PR TITLE
Add Windows builds

### PR DESCRIPTION
**Pull request by kokaz**
This adds Windows binary to GitHub releases

### CODE REVIEW

**Comment by akupila** [comment]
Do we need to provide 32bit windows builds? Is that still a thing

File: wercker.yml [hunk 1]
```diff
@@ -37,6 +37,17 @@ build:
             cp $WERCKER_OUTPUT_DIR/artifacts/latest/darwin_amd64/stern $WERCKER_REPORT_ARTIFACTS_DIR
 
     - script:
+        name: go build windows amd64
+        code: |
+            export CGO_ENABLED=0
+            GOOS=windows GOARCH=amd64 go build \
+                -ldflags="-s -X github.com/wercker/stern.GitCommit=$WERCKER_GIT_COMMIT -X github.com/wercker/stern.PatchVersion=$(( ($(date +%s) - $(date --date=20150101 +%s) )/(60*60*24) )) -X github.com/wercker/stern.Compiled=$(date +%s)" \
+                -installsuffix cgo \
+                -o $WERCKER_OUTPUT_DIR/artifacts/latest/windows_amd64/stern
```
Comment (akupila) [inline wercker.yml]: should this be `stern.exe`?

Comment (kokaz) [inline wercker.yml]: It is! my bad :)

**Comment by kokaz** [comment]
@akupila I don't think 32 bits support worth it, if people complain we will add it but I'm sure no one will

File: wercker.yml [hunk 2]
```diff
@@ -83,3 +94,9 @@ release-stable:
         file: ./artifacts/latest/darwin_amd64/stern
         filename: stern_darwin_amd64
         content-type: application/octet-stream
+
+    - github-upload-asset:
+        token: $GITHUB_TOKEN
+        file: ./artifacts/latest/windows_amd64/stern
+        filename: stern_windows_amd64
```
Comment (flenter) [inline wercker.yml]: Wouldn't it be better if it had a file extension like `.exe`?

Comment (kokaz) [inline wercker.yml]: @flenter Good catch, fixed!

File: wercker.yml (hunk 1 above)
Comment (kokaz) [inline wercker.yml]: @akupila Fixed!

**Pull request merged**